simpleclient/src/main/java/io/prometheus/client/Counter.java

```diff
@@ -73,41 +81,83 @@
  */
 public class Counter extends SimpleCollector<Counter.Child> implements Collector.Describable {
 
+  private final Boolean exemplarsEnabled; // null means default from ExemplarConfig applies
+  private final CounterExemplarSampler exemplarSampler;
+
   Counter(Builder b) {
     super(b);
+    this.exemplarsEnabled = b.exemplarsEnabled;
+    this.exemplarSampler = b.exemplarSampler;
+    initializeNoLabelsChild();
   }
 
   public static class Builder extends SimpleCollector.Builder<Builder, Counter> {
+
+    private Boolean exemplarsEnabled = null;
+    private CounterExemplarSampler exemplarSampler = null;
+
     @Override
     public Counter create() {
       // Gracefully handle pre-OpenMetrics counters.
       if (name.endsWith("_total")) {
         name = name.substring(0, name.length() - 6);
       }
+      dontInitializeNoLabelsChild = true;
       return new Counter(this);
     }
+
+    /**
+     * Enable exemplars and provide a custom {@link CounterExemplarSampler}.
+     */
+    public Builder withExemplarSampler(CounterExemplarSampler exemplarSampler) {
+      if (exemplarSampler == null) {
+        throw new NullPointerException();
+      }
+      this.exemplarSampler = exemplarSampler;
+      return withExemplars();
+    }
+
+    /**
+     * Enable exemplars.
+     * <p>
+     * This will enable exemplars for this counter even if exemplars are disabled by default in {@link ExemplarConfig}.
+     */
+    public Builder withExemplars() {
+      this.exemplarsEnabled = TRUE;
+      return this;
+    }
+
+    /**
+     * Disable exemplars.
+     * <p>
+     * This will disable exemplars for this counter even if exemplars are enabled by default in {@link ExemplarConfig}.
```

Comment:
This reads as though it prevents all use of exemplars, whereas it's only the automatic ones that it stops. This could be clearer.

Reply:
Changed the wording.